src/Main.java

```diff
@@ -1,5 +1,72 @@
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    // метод для задачи 1
+    public static void defineYear(int year) {
+        String msg1 = "%d год является високосным \n";
+        String msg2 = "%d год не является високосным \n";
+        if (year < 1584) {
+            System.out.println("Високосный год был введен в 1584, поэтому все предшествующие года не могут считаться високосными или не високосными.");
+        } else if (year % 4 != 0) {
+            System.out.printf(msg2, year);
+        } else if (year % 100 == 0 && year % 400 != 0) {
+            System.out.printf(msg2, year);
+        } else {
+            System.out.printf(msg1, year);
+        }
+    }
+
+    // Метод для задачи 2
+    public static void defineOS(byte oS, int year) {
+        if (year >= 2015) {
+            switch (oS) {
+                case 0: System.out.println("Установите версию приложения для iOS по ссылке");
+                    break;
+                case 1: System.out.println("Установите версию приложения для Android по ссылке");
+                    break;
+            }
+        } else if (year < 2015) {
+            switch (oS) {
+                case 0:
+                    System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+                    break;
+                case 1:
+                    System.out.println("Установите облегченную версию приложения для Android по ссылке");
+                    break;
+            }
+        }
+    }
+
+// Метод для задачи 3
+       public static int defineDay(int i){
+           int day = 0;
+           if (i < 20) {
+               day += 1;
+           } else if (i >= 20 && i < 60) day = day + 2;
+           else if (i >= 60 && i < 100) day = day + 3;
```

Comment:
```suggestion
           else if (i >= 60 && i < 100) {
               day = day + 3;
           }
```
фигурные скобки надо всегда писать, даже если 1 строка всего